packages/node/src/lib/customhttpclient.ts

```diff
@@ -0,0 +1,20 @@
+export interface CustomHTTPClient {
+  send: (resource: any, options: any) => Promise<Response>
+}
+
+export class DefaultFetchClient implements CustomHTTPClient {
```

Comment:
Sounds a little odd to me... I think this should be called `DefaultHttpClient`

Reply:
Fixed